lib/amperize.js

```diff
@@ -32,6 +32,8 @@ var DEFAULTS = {
   }
 };
 
+var called;
```

Comment:
This variable needs to live inside of the function scope i think. It needs to be present for each call and not as global value.